walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,76 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
-  }
-}
+    public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+        Collection<TimeRange> timeRanges = new ArrayList<TimeRange>();
+        List<TimeRange> attendeeRelevantTimeRanges = new ArrayList<TimeRange>();
+        Collection<String> requestAttendees = request.getAttendees(); 
+        Collection<String> eventAttendees;
```

Comment:
Since these are only ever used in the for loop, declaring it right inline with the for loop helps organization. (aka you can just call `.disjoint(request.getAttendees(), event.getAttendees())`)

Reply:
Got it.